src/hooks/use-assistant.ts

```diff
@@ -14,47 +14,48 @@ export type Message = {
 	createdAt: number; // Unix timestamp
 };
 
-const chatIdStoreKey = ( instanceId: string ) => `ai_chat_id_${ instanceId }`;
-const chatMessagesStoreKey = ( instanceId: string ) => `ai_chat_messages_${ instanceId }`;
+export type MessageDict = { [ key: string ]: Message[] };
```

Comment:
I am guessing that MessageDict stands for something like Message Dictionary?

Reply:
Yes, do you want me to rename it, to make it more clear?

Reply:
Personally it was not immediately clear to me what it was but it might be my lack of experience rather than a bad naming itself. Let's maybe wait for someone to weight in and hear for another opinion to see if it is clear enough

Reply:
I've seen often the usage of the word `Dict` as the suffix in variables, especially in iOS development. So from my side, it seems clear to me that it's a dictionary. In any case, I'd like to note I'm not against renaming it for a clearer value.